### PR TITLE
first pull request

### PR DESCRIPTION
**Pull request by sambogaert**
@kapunahelewong my first ever pull request on a public repo! 😄

### CODE REVIEW

**Comment by kapunahelewong** [comment]
This is great, @sambogaert! I'll, add some labels, comment, and merge. If you'd like me to do some other things, just let me know. You can submit as many PRs or issues as you'd like (or even just comment on them). Thank you!! 😊

File: pullrequests/index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h2>This is your first pull request</h2>
 	</ul>
 
 	<p>Below this p tag there is another p tag with the name and date of the last author. Change it for your own.</p>
-	<p>Cutiko, June 26 2019</p>
+	<p>Sam, July 29 2019</p>
```
Comment (kapunahelewong) [inline pullrequests/index.html:24]: I'm reviewing just to review, so visitors can see what it's like. I added this comment by clicking on line 24.

**Review by kapunahelewong — APPROVED** [review]
This is the review. (Everything is perfect of course!)

File: pullrequests/index.html (hunk 1 above)
Comment (kapunahelewong) [reply pullrequests/index.html:24]: I'll just merge, but if you were to make edits due to a request in this comment, you'd then respond with something like, "Done", Fixed", or a question here, etc. Then, when it was done, you could click "Resolve conversation". I tend to resolve mine whenever I can on the Angular repo so people know I addressed the comment. It also neatly collapses them. 

The conversations here can prove super helpful later, especially for large, long-lived PRs.

**Comment by kapunahelewong** [comment]
Merged! 💃🏻

Notice that you can continue to have conversations in merged PRs. I'm not using a rebased workflow for merging (like angular), but if I were, you'd see in this PR "closed with unmerged commits", and it would just look closed. However, if you looked closely, you'd see it was closed via some hash number. Clicking that hash would take you to a page where you could see that it had indeed been merged. Here's an example:

https://github.com/angular/angular/pull/28571#event-2500416138

That repo also has a bot called Mary Poppins that also renders the UI so people can view the changes, so that's what some of those extra comments are.

Anyway, just some thoughts about PRs. 😊

File: pullrequests/index.html (hunk 1 above)
Comment (sambogaert) [reply pullrequests/index.html:24]: Awesome!

**Comment by sambogaert** [comment]
Wow, thanks for your time and the awesome info! Very helpfull! 🙏 
I'll definitely play around a bit more and try some things out. 
My very first steps in the OSS world are made thanks to your repo, thank you for that! ☺️

**Comment by kapunahelewong** [comment]
aw, that's great to hear, @sambogaert! Please, experiment freely. That's the whole purpose of this repo. If you have questions, feel free to tag me or hit me up on twitter (@kapunahele). Thanks for hanging out and trying stuff. 🌸

**Pull request merged**